src/derive.rs

```diff
@@ -33,66 +35,143 @@ declare_lint! {
     "deriving `Hash` but implementing `PartialEq` explicitly"
 }
 
+/// **What it does:** This lint warns about explicit `Clone` implementation for `Copy` types.
+///
+/// **Why is this bad?** To avoid surprising behaviour, these traits must agree and the behaviour
```

Comment:
s/must/should/, s/cannot/should not/, s/override/overridden.

Mention that in almost all situations a `Copy` type should have a `Clone` impl that does nothing more than copy the object, which is what `#[derive(Copy, Clone)]` gets you.